packages/xml-views-validation/src/api.ts

```diff
@@ -42,6 +42,9 @@ export function validateXMLView<ExternalXMLViewIssue>(opts: {
   const validatorVisitor = new ValidatorVisitor(opts.context, opts.validators);
   accept(opts.xmlView, validatorVisitor);
   const issues = validatorVisitor.collectedIssues;
+
+  const nonUniqueIdIssues = validateNonUniqueID(opts.xmlView, opts.context);
```

Comment:
We still revalidate all the documents multiple times here. We should only do it once.

Reply:
We validate once per file. But now number files all opened documents. See above for more details.